tests/DataOverlayMenu.spec.ts

```diff
@@ -13,6 +13,8 @@ test('should display added, selected and removed segmentation promptly', async (
   await page.getByText('SELECT A SEGMENTATION').click();
   await page.getByTestId('2d-tta_nnU-Net_Segmentation').click();
 
+  await page.waitForTimeout(5000);
```

Comment:
Slowly adding back in all the timeouts I once removed. Many screenshots had to be replaced because they incorrectly had the LOAD button - now it is gone!